doc/getting-started.md

```diff
@@ -44,10 +44,79 @@ You now have four servers, running on localhost:31410, 31430, 31450 and 31470. T
 
 You are now done. Everything below this line is extra information for the incurably curious.
 
-### important notes for the admin server > widget management pages
+### Important notes for the admin server > widget management pages
 Each widget management page shows a preview of how the widget will look given a set of configurations.
 To make this work however, one must first use the command "npm run build" in the specific widget (in the packages > [widget] folder). This is only neccesary in development. In production these widgets are already prebuild.
 
+### Getting the widget preview to work:
+There are a few steps to follow when trying to get a new widget rendered for the preview component:
+* In widget-settings.js define the way the api-server should bundle the widget. For example:
+  ```
+  resourceoverview: {
+    js: ['@openstad-headless/resource-overview/dist/resource-overview.iife.js'],
+    css: ['@openstad-headless/resource-overview/dist/style.css'],
+    functionName: 'OpenstadHeadlessResourceOverview',
+    componentName: 'ResourceOverview',
+    defaultConfig: {
+      projectId: null,
+    },
+  }
+  ```
+* In widget-preview.tsx you will need to make sure that the widget-type is in the [type] key of the Props definition. This will be the key how the widget-preview route in the api-server decides which and how the widget should be packaged in a .js script.
```

Comment:
In widget-preview.tsx, you will need to make sure that the widget-type is defined in the [type] key of the Props definition. This will be the key that the widget-preview route in the api-server uses to decide how the widget should be packaged, through use of a .js script.